packages/dredd-transactions/README.md

```diff
@@ -94,95 +86,91 @@ Result of compilation. Alongside compiled [Transaction][transaction-object-spec]
-        - (object)
-            - name: `Content-Type` (string)
-            - value: `text/plain` (string)
-    - body: `Hello world!\n` (string)
+  - method
+  - uri: `/message` (string) - Informative URI of the Request.
+  - headers (array) - List of HTTP headers in their original order, with the original casing of the header name, including multiple headers of the same name.
+    - (object)
+      - name: `Content-Type` (string)
+      - value: `text/plain` (string)
+  - body: `Hello world!\n` (string)
 - response (object) - Expected HTTP Response as described in API description document.
-    - status: `200` (string)
-    - headers (array) - List of HTTP headers in their original order, with the original casing of the header name, including multiple headers of the same name.
-        - (object)
-            - name: `Content-Type` (string)
-            - value: `text/plain` (string)
-    - body (string, optional)
-    - schema (string, optional)
-
+  - status: `200` (string)
+  - headers (array) - List of HTTP headers in their original order, with the original casing of the header name, including multiple headers of the same name.
+    - (object)
+      - name: `Content-Type` (string)
+      - value: `text/plain` (string)
+  - body (string, optional)
+  - schema (string, optional)
 
 ### Deprecated Properties
 
 - name: `Hello world! > Retrieve Message` (string) - Transaction Name, non-deterministic breadcrumb location of the HTTP Transaction within the API description document.
 - origin (object) - Object of references to nodes of [API Elements][api-elements] derived from the original API description document.
-    - filename: `./api-description.apib` (string)
-    - apiName: `My Api` (string)
-    - resourceGroupName: `Greetings` (string)
-    - resourceName: `Hello, world!` (string)
-    - actionName: `Retrieve Message` (string)
-    - exampleName: `First example` (string)
+  - filename: `./api-description.apib` (string)
+  - apiName: `My Api` (string)
+  - resourceGroupName: `Greetings` (string)
+  - resourceName: `Hello, world!` (string)
+  - actionName: `Retrieve Message` (string)
+  - exampleName: `First example` (string)
 
 > **Note:** These properties are to be superseded by whatever comes out of the proposal in [apiaryio/dredd#227](https://github.com/apiaryio/dredd/issues/227).
 
-
 <a name="annotation-object"></a>
+
 ### Annotation (object)
 
 Description of an error or warning which occurred during parsing of the API description or during compilation of transactions.
 
 #### Properties
 
 - type (enum[string])
-    - `error`
-    - `warning`
+  - `error`
+  - `warning`
 - component (enum[string]) - In which component of the compilation process the annotation occurred
-    - `apiDescriptionParser`
-    - `parametersValidation`
-    - `uriTemplateExpansion`
+  - `apiDescriptionParser`
```

Comment:
To avoid quirks of various Markdown parsers and various random problems, it's usually the best to stick to indenting with four spaces.

<img width="473" alt="image" src="https://user-images.githubusercontent.com/283441/66581929-3ee4c780-eb81-11e9-81ac-140130eef2b8.png">

https://daringfireball.net/projects/markdown/syntax

Reply:
The point is taken 👍I'm confused about "one tab" part, as tab width may vary. I.e. I always have 2 spaces as 1 tab. Will revert in the commit that added the dredd-transactions package.

Reply:
Tab here is understood as the tab character, not as tab translated to a number of spaces. Resulting tab length can vary for the reader depending on settings of the editor where you have the file open, but it is unambiguous for the writer and the parser - it's always one `\t` character.